plot/plot_lmbench

```diff
@@ -0,0 +1,51 @@
+#! /bin/bash
+
+plot() {
```

Comment:
help move { to next line

Reply:
file submitted as another PR: https://github.com/intel/lkp-tests/pull/162